src/utils/packageUtils.ts

```diff
@@ -510,7 +517,10 @@ export function copyDescriptorProperties(
     packageDescriptorJsonCopy,
     Object.fromEntries(
       ['country', 'edition', 'language', 'features', 'orgPreferences', 'snapshot', 'release', 'sourceOrg'].map(
-        (prop) => [[prop], definitionFileJsonCopy[prop]]
+        (prop) => {
+          const matchCase = Object.keys(definitionFileJsonCopy).find((key) => key.toLowerCase() === prop.toLowerCase());
+          return [[prop], matchCase ? definitionFileJsonCopy[matchCase] : undefined];
+        }
```

Comment:
This should be equivalent code as line 522 since an object lookup that doesn't resolve to anything is `undefined`, including if `matchCase` is `undefined`.
`return [[prop], definitionFileJsonCopy[matchCase]];`

Reply:
@shetzel doing that change throws an error - `Type 'undefined' cannot be used as an index type.ts(2538)`

Another option is to set the default of `undefined`
`return [[prop], definitionFileJsonCopy[matchCase ?? 'undefined']];`

Reply:
Ok that works then.  Thanks!